sources/models/src/aws-k8s-1.21/defaults.d/60-cf-signal.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/cf-signal.toml
```

Comment:
nit: "60" overlaps with the lockdown config. We could make this "25" rather than "60", to group it with `20-aws-host-containers.toml`, or else make it "80" to start a new group of settings.